src/Symfony/Component/HttpKernel/Controller/ContainerControllerResolver.php

```diff
@@ -86,6 +94,18 @@ protected function instantiateController($class)
             return $this->container->get($class);
         }
 
-        return parent::instantiateController($class);
+        $e = null;
```

Comment:
Could be removed and use `isset($e)` in if below ?

Reply:
isn't better for performance to do this  instead of isset ?

Reply:
Hmm, no idea. Actually I though we already used `isset($e)` for such cases in many places. But it seems I was wrong. So fine 👍

Reply:
Setting to null is the CS we use in the code base.

Reply:
oh, in fact, this can be removed, the variable is not needed: if we reach L99, $e will always be non null
which means the null !== $e check can be removed from the below if also